config_defaults.json

```diff
@@ -105,8 +105,8 @@
     "bindAddress": "0.0.0.0:8081",
     "basicAuth": {
       "enabled": false,
-      "username": "core",
-      "password": "core"
+      "username": "iota-core",
```

Comment:
you have to change these values in the `params.go` of the dashboard component, and then run `tools/gendoc` instead of modifying the file manually.